rudder-web/src/main/scala/com/normation/rudder/web/model/DirectiveFieldEditors.scala

```diff
@@ -176,10 +175,10 @@ class SelectField(val id: String, items: Seq[ValueLabel]) extends DirectiveField
     val valuesByLabel = items.map(_.reverse.tuple).toMap
     val labelsByValue = items.map(_.tuple).toMap
 
-    val defaultLabels = values.map(labelsByValue(_))
+    val defaultLabels = values.map(labelsByValue.getOrElse(_,""))
```

Comment:
Maybe we need a logger warning here...

I think invalid values should be handled before and should not arrive at this point